openmc_plotter/plotmodel.py

```diff
@@ -467,10 +504,32 @@ def _do_op(array, tally_value, ax=0):
 
         return image_data, None, data_min, data_max
 
+    def _create_distribcell_image(self, tally, tally_value, scores, nuclides):
+        sp = self.statepoint
+        dfilter = tally.find_filter(openmc.DistribcellFilter)
+
+        data = tally.get_values(scores=scores, nuclides=nuclides, value=tally_value)
+        data = data.flatten()
+
+        cell_id = dfilter.bins[0]
+        # create a mask for ids that match the cell
+        image_data = np.zeros_like(self.ids)
```

Comment:
I tried out this branch and it wasn't working for me; eventually traced it down to this!
```suggestion
        image_data = np.zeros_like(self.ids, dtype=float)
```

Also, for a distribcell tally, should we not set the initial state to all `np.nan` instead of all zeros? It doesn't really make sense to me to assign zero values to locations in the image data where we don't have matching bins.